AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithUserPoolIntegrationTests/GraphQLWithUserPoolIntegrationTests.swift

```diff
@@ -169,10 +169,10 @@ class GraphQLWithUserPoolIntegrationTests: XCTestCase {
         let completeInvoked = expectation(description: "request completed")
         let uuid = UUID().uuidString
 
-        // create a Todo mutation with a missing/invalid "description" variable value
+        // create a Todo mutation with a missing/invalid "name" variable value
```

Comment:
the default Todo schema when generating the API has required name and optional description. This wasn't the case before.. I found it easier to maintain the steps in the provisioning as is (use default Todo schema, while displaying which schema was used), so the test code now needs to be updated to accomdate the new schema.